apps/site/assets/ts/stop/components/AccessAmenitiesModal.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { Alert, Facility } from "../../__v3api";
+import Alerts from "../../components/Alerts";
+import { hasFacilityAlert } from "../../models/alert";
+import { AmenityModal, AmenityLink } from "./amenities/AmenityCard";
+
+const AccessAmenitiesModal = ({
+  stopName,
+  alerts,
+  facilities,
+  facilityType
+}: {
+  stopName: string;
+  alerts: Alert[];
+  facilities: Facility[];
+  facilityType: "Elevators" | "Escalators";
+}): JSX.Element => {
+  const hasFacilities = facilities ? facilities.length > 0 : false;
+  const linkText =
+    facilityType === "Elevators"
+      ? "Report an elevator issue"
+      : "Report an escalator issue";
+  return (
+    <>
+      {hasFacilities && (
+        <AmenityModal headerText={`${facilityType} at ${stopName}`}>
+          <Alerts alerts={alerts} />
+          <h2>Elevator Status</h2>
```

Comment:
This should probably be `<h2>{facilityType} Status</h2>` to make it work with both Elevators and Escalators